src/Shopsys/ShopBundle/Resources/translations/messages.de.po

```diff
@@ -2,884 +2,868 @@ msgid ""
 msgstr ""
```

Comment:
Why have the file permissions changed (`100644 → 100755`)?

Reply:
fixed